elements/urth-viz-explorer/urth-viz-explorer.html

```diff
@@ -56,6 +56,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bars in barchart, slices in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
No need for property here once it is moved to `elements/urth-viz-chart/urth-viz-chart-common-behavior.html`.